client/list.py

```diff
@@ -1,8 +1,21 @@
+from email import header
```

Comment:
can you provide context to why this import is required ? it doesn't seem to be used.

Reply:
@nukoaka this has been fixed in this commit e6abfe8269e001446f6580a7647bf729d190d8b9